ecs-init/exec/iptables/iptables.go

```diff
@@ -48,14 +48,15 @@ const (
 	iptablesTableFilter = "filter"
 	iptablesTableNat    = "nat"
 
-	offhostIntrospectionAccessConfigEnv   = "ECS_ALLOW_OFFHOST_INTROSPECTION_ACCESS"
-	offhostIntrospectonAccessInterfaceEnv = "ECS_OFFHOST_INTROSPECTION_INTERFACE_NAME"
```

Comment:
Is this env var not plugged in anywhere? If not, we should remove it from [readme](https://github.com/aws/amazon-ecs-agent/blob/37ff4978288962e1f1796472a176e84c46707c56/README.md?plain=1#L267)